README.md

```diff
@@ -197,6 +197,9 @@ Example:
 
 > **Note**: `pr` will automatically create the GitHub PR for you and add the appropriate `Ready to Merge into Develop` label
 
+You can also provide a `--l10n` flag to the command and it will create a PR against your base branch
```

Comment:
Let's discuss with @elijahmanor and come up with a more general purpose flag. Is there a reason this can't be the new default, or perhaps just ask "Didn't find a bump commit, do you want to PR anyway?"

Reply:
We talked about it a bit. I could change the flag to be something like `--no-packages` or `--no-pkgs`, etc... Currently, the `pr` command is too deeply reliant on there being packages to bump that it would take a lot of work to make it work without them so a flag is really the best option we have.

Reply:
I lean toward something like `tag-release pr --no-bump` since it's more generic and not specific to localization. It's a useful feature with or without l10n changes